packages/fast-sketch-library/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@microsoft/fast-sketch-library",
+  "description": "A Node.js utility for quickly extracting Sketch symbol libraries from a website",
+  "version": "1.0.0",
+  "author": {
+    "name": "Microsoft",
+    "email": "fwq@microsoft.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/fast-dna.git"
+  },
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "clean:dist": "node ../../build/clean.js dist",
+    "prepare": "npm run clean:dist && tsc -p ./tsconfig.json && webpack",
+    "test": "npm run tslint",
+    "tslint": "tslint -c ./tslint.json 'src/**/*.ts{,x}' '**/*.ts{,x}'",
+    "tslint:fix": "tslint -c ./tslint.json --fix 'src/**/*.ts{,x}' '**/*.ts{,x}'"
+  },
+  "license": "MIT",
+  "readme": "README.md",
+  "devDependencies": {
+    "@brainly/html-sketchapp": "^3.1.0",
+    "@microsoft/fast-tslint-rules": "^1.6.0",
```

Comment:
technically at 1.7 if you want to change it.

Reply:
updated.